client/app/core/navigation/navigation-controller.js

```diff
@@ -60,6 +60,7 @@ export function NavigationController (Text, Navigation, Session, API_BASE, Shopp
       helpMenu: RBAC.hasAny(['about', 'product', 'documentation'])
     }
     EventNotifications.setToastDisplay(vm.permissions.suiNotifications)
+    const appBasePath = process.env.NODE_ENV === 'production' ? '' : 'assets/'
```

Comment:
How have you verified this in production?

Reply:
we tested locally by running `yarn build`